WrightTools/artists/_base.py

```diff
@@ -332,50 +332,28 @@ def legend(self, *args, **kwargs):
             kwargs["framealpha"] = 1.
         return super().legend(*args, **kwargs)
 
-    def pcolor(self, *args, **kwargs):
-        """Create a pseudocolor plot of a 2-D array.
-
-        Uses pcolor_helper to ensure that color boundaries are drawn
-        bisecting point positions, when possible.
-
-        Parameters
-        ----------
-        data : 2D WrightTools.data.Data object
-            Data to plot.
-        channel : int or string (optional)
-            Channel index or name. Default is 0.
-        dynamic_range : boolean (optional)
-            Force plotting of all contours, overloading for major extent. Only applies to signed
-            data. Default is False.
-        autolabel : {'none', 'both', 'x', 'y'}  (optional)
-            Parameterize application of labels directly from data object. Default is none.
-        xlabel : string (optional)
-            xlabel. Default is None.
-        ylabel : string (optional)
-            ylabel. Default is None.
-        **kwargs
-            matplotlib.axes.Axes.pcolor__ optional keyword arguments.
-
-            __ https://matplotlib.org/api/_as_gen/matplotlib.axes.Axes.pcolor.html
-
-        Returns
-        -------
-        matplotlib.collections.PolyCollection
-        """
+    def _parse_plot_args(self, *args, **kwargs):
```

Comment:
This merges the duplicitous code between pcolor and pcolormesh.  It seems like all our other plotting commands could be sent here as well to further simplify, but perhaps I make an issue for that and address those concerns with another PR.

Reply:
I think I agree that it should hold off to send others here, but it's a good start. Thanks